### DISCUSSION TITLE
In metrics coredns_dns_request_duration_seconds_bucket absent labels for additional server and zone

### DISCUSSION THREAD

**Discussion by osipovdaniil** [coredns/coredns#5797] [Q&A] [answered]
Hello! 
CoreDNS version `1.6.2`

I have additional zones in config coreDNS:
```
Corefile: |
  consul-sql-staging.my.domain {
    errors
    forward . 10.10.10.10
  }
  consul-sql.my.domain {
    errors
    forward . 11.11.11.11
 }
```

I want see latency per zone in metrics `coredns_dns_request_duration_seconds_bucket`
But i got only `{server="dns://:53", zone="."}`

<img width="1789" alt="Снимок экрана 2022-12-08 в 14 01 37" src="https://user-images.githubusercontent.com/73994399/206430473-b0246487-c104-4643-a559-2a9069ade075.png">

How do I get metrics for `{server="dns://:53", zone=".consul-sql-staging.my.domain"}` ?

**Comment by Tantalor93** [comment, marked as answer]
Hello, have you tried enabling `prometheus` CoreDNS plugin in given server blocks? https://github.com/coredns/coredns/tree/master/plugin/metrics

**Reply by Tantalor93** [reply]
```
Corefile: |
  consul-sql-staging.my.domain {
    errors
    forward . 10.10.10.10
	prometheus localhost:9253
  }
  consul-sql.my.domain {
    errors
    forward . 11.11.11.11
	prometheus localhost:9253
 }
```

**Reply by osipovdaniil** [reply]
Yes, prometheus plugin is enabled

<img width="477" alt="Снимок экрана 2022-12-08 в 14 15 26" src="https://user-images.githubusercontent.com/73994399/206433226-c227fbd8-7a52-4ec7-9320-74b500738c70.png">

**Reply by Tantalor93** [reply]
see my example, the plugin needs to be declared in each server block

**Reply by osipovdaniil** [reply]
oh I understand

**Reply by osipovdaniil** [reply]
yes, ok thanks
I will try to do this
